tycho-surefire/tycho-surefire-plugin/src/main/java/org/eclipse/tycho/surefire/provisioning/ProvisionedInstallationBuilder.java

```diff
@@ -126,7 +126,7 @@ private void executeDirector(TargetEnvironment env) throws MojoFailureException
         for (String iu : ius) {
             command.addUnitToInstall(iu);
         }
-        command.setDestination(DirectorRuntime.getDestination(effectiveDestination, env));
+        command.setDestination(effectiveDestination);
```

Comment:
This now no longer takes the environment into account.

Reply:
This is considered, (the effective destination already contains the Contents/Eclipse stuff.

The actual problem is here now:
`work` in `tycho-surefire-plugin`
https://github.com/sratz/tycho/blob/361fd9881e7bbc58046ea3c791fb5679cb1e6f2e/tycho-its/projects/surefire.p2InstalledRuntime/extProductTestDirector/pom.xml#L52
behav
is now interpreted differently than
`destination` in `tycho-p2-director-plugin`
https://github.com/sratz/tycho/blob/361fd9881e7bbc58046ea3c791fb5679cb1e6f2e/tycho-its/projects/surefire.p2InstalledRuntime/extProductTestDirector/pom.xml#L62

and the 'magic' is now not consistent anymore :(

I now get your pint in the issue's discussion better now.

This `Contents/Eclipse` layout on mac is real mess and makes things really complicated.

I'll have to think about this some more.

Reply:
@sratz maybe Tycho can reuse the "logic" from the Director somehow so they behave equally?